.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```diff
@@ -0,0 +1,108 @@
+name: "Report a Linux Issue"
+description: "Create a report for a runtime related Linux Issue"
+labels: ["os:linux"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv version
```

Comment:
I would split off mpv version to own box and instruct to provide `mpv --version` output. It includes more information than only a version and `0.38` means not much if the issue might be in dependencies.